xio-core/src/main/resources/reference.conf

```diff
@@ -163,6 +163,8 @@ xio {
     remotePort = 0
     localIp = ""
     localPort = 0
+    idleTimeoutEnabled = false
+    idleTimeoutDuration = 0
```

Comment:
At some point we may want the type of incoming connection to determine if there’s no idle timeout. This is fine for now.

Reply:
k